client/search_box.html

```diff
@@ -0,0 +1,12 @@
+<template name="search_box">
+  <div class="form-group">
+          <div class="col-xs-5 selectContainer">
+              <select name="invoice_field_menu" class="form-control">
+                  <option value="0">Email</option>
+                  <option value="1">Invoice Number</option>
+                  <option value="2">Total</option>
```

Comment:
It's probably better to name the values accordingly to their choices, e.g: `<option value="email">Email</option>`